play/src/front/Components/Menu/SettingsSubMenu.svelte

```diff
@@ -555,38 +578,23 @@
                 </div>
             </div>
         </div>
+
         <label for="cam-toggle" class="flex cursor-pointer items-center relative m-4">
-            <input
-                type="checkbox"
+            <InputSwitch
```

Comment:
For all the InputSwitch in this file, you should remove the &lt;label> tag above (or replace it with a div if needed).
The `InputSwitch` component already contains a label.